.github/workflows/ci.yml

```diff
@@ -1,23 +1,23 @@
 name: Etcd Client Test
 run-name: ${{ github.actor }} is testing out etcd client
 on:
-  pull_request_target:
-    branches: ['**']
+  pull_request:
```

Comment:
`pull_request_target` not only means use the workflow file in the base branch, but actually all the code in the base branch, so we don't test the pull request actually.